service/block/event.go

```diff
@@ -72,6 +72,12 @@ func (s *Service) handleRawBlock(raw *RawBlock) error {
 			"block hash", raw.Hash().String())
 		return err
 	}
-	// TODO @renaynay: store extended block
+	err = s.StoreBlockData(context.Background(), extendedBlockData)
+	if err != nil {
+		log.Errorw("storing block", "err msg", err, "block height", raw.Height, "block hash",
+			raw.Hash().String())
+		return err
+	}
+	log.Infow("storing block", "block height", raw.Height, "block hash", raw.Hash().String())
```

Comment:
nit: isn't the block already stored at this point? Or does it happen aysnc in the background?
```suggestion
	log.Infow("stored block", "block height", raw.Height, "block hash", raw.Hash().String())
```